### ISSUE TITLE
Separate JMP profiles for different users

### ISSUE THREAD

**Issue by ftc2** [jellyfin/jellyfin-media-player#333]
is there some kind of a way (even a hacky way) to launch JMP with different login/server information?

on a media center windows pc, what i want is to have separate shortcuts to JMP -- each with a different family member's profile already logged in -- so each user can quickly and conveniently access his own watch progress etc in JF. otherwise we have to log out and log in each time in JF which is pretty inconvenient.

is there some JMP flag that can manually specify the settings path? or some way to make JMP "portable"?

the only thing i can think of is to have a windows account for each user, but that is not really any more convenient.

thanks

**Comment by NaturalBornCamper** [comment]
Same goes for the AppleTV app, also wondering if there is such a way coming up in the future for other family members, in a similar way to what Netflix does.

**Comment by iwalton3** [comment]
> otherwise we have to log out and log in each time in JF which is pretty inconvenient.

Yes that is an issue across most of the Jellyfin applications. I don't have a great solution for that. It doesn't help that any "correct" way to do this is defeated by the fact that the Jellyfin server only allows a client to be logged in as a single user at a time, unless it pretends to be multiple separate clients.
